.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,151 @@
+---
+name: Bug Report
+description: Is something not working as expected? Submit a bug.
+labels:
+  - [bug, needs-triage]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Before filing an issue, please [search the existing issues](https://github.com/vmware/terraform-provider-nsxt/issues?q=is%3Aissue+is%3Aopen+label%3Abug) (open or closed), and use the [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) feature to add up-votes to existing issues.
+
+        When filing an issue, please include the following information.
+  - type: checkboxes
+    id: terms
+    attributes:
+      label: Code of Conduct
+      description: The Code of Conduct applies to you when interacting with the community here on GitHub and contributing code.
+      options:
+        - label: I have read and agree to the [Code of Conduct](https://github.com/vmware/terraform-provider-nsxt/blob/main/CODE_OF_CONDUCT.md).
+          required: true
+        - label: Vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue initial description to help the maintainers prioritize.
```

Comment:
Can we have lines 21-26 in a markdown section as for the documentation template?

Reply:
Resolved in https://github.com/vmware/terraform-provider-nsxt/commit/50fcb7ad1539140b4f6f4a14864b57ee36196e59.